qubesmanager/table_widgets.py

```diff
@@ -281,26 +264,33 @@ def update_vm_state(self):
         self.upd_info.update_outdated()
 
 
-class VmTemplateItem(QtGui.QTableWidgetItem):
-    def __init__(self, vm):
-        super(VmTemplateItem, self).__init__()
+class VMPropertyItem(QtGui.QTableWidgetItem):
+    def __init__(self, vm, property_name, empty_function=(lambda x: False),
+                 check_default=False):
```

Comment:
This could use some docstring describing arguments. What `empty_function` should do (and with what arguments it's called)? Same for `check_default`?